### ISSUE TITLE
stuck at respond/create 

### ISSUE THREAD

**Issue by awuworker** [madoublet/respond#488]
I already installed it locally and have a virtual host www/repond/app/public

I am stuck at creating a site, it always goes to error - js prompt of error logo.

Need some assistance on this. Thanks in advance

**Comment by axis80** [comment]
I don't understand what you mean by "it always goes to error - js prompt of error logo."  Can you post a screenshot or something?  If you press Ctrl-Shift-J to open the browser console, what error messages are listed there?

**Comment by awuworker** [comment]
![image](https://cloud.githubusercontent.com/assets/22208354/21949091/774077c0-da2a-11e6-833f-8903b9ba5aad.png)

**Comment by awuworker** [comment]
![image](https://cloud.githubusercontent.com/assets/22208354/21949111/a1b19eee-da2a-11e6-9368-7a311bcebe84.png)

**Comment by axis80** [comment]
I ran into this same problem a few weeks ago, but I can't remember what I did to fix it.  I think maybe I was missing a necessary PHP module, and once installed that it started working?

Anyway, I think the next things to check here are the error logs on the server.  Look in storage/logs/lumen.log (within your webroot) and also in the Apache error_log for that site.  Let me know what errors you see there?
